contrib/devtools/security-check.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2
+#!/usr/bin/env python2
```

Comment:
I made no attempt to check for portability of the python scripts between python releases, just a straight s/// on all *.py files